contribute.html

```diff
@@ -0,0 +1,271 @@
+</head>
+
+<body>
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top">
+        <a class="navbar-brand" href="index.html">
+            <img src="img/global/loopback-full-logo-blue.svg" width=150px>
+        </a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+            aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link" href="getting-started.html">Getting Started</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="resources.html">Resources</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link highlighted" href="contribute.html">Contribute</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="http://loopback.io/doc/en/lb4">Docs</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="oasgraph.html">GraphQL</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="https://ibm.biz/apitest">Test Your APIs</a>
+                </li>
+                <li class="nav-item" style="padding-top: 10px">
+                    <a class="github-button" href="https://github.com/strongloop/loopback-next" data-icon="octicon-star" data-show-count="true"
+                        aria-label="Star strongloop/loopback-next on GitHub">Star</a>
+                </li>
+            </ul>
+        </div>
+    </nav> 
+
+    <div class="r-masthead">
+        <div class="container-fluid"></div>
+        <div class="header-space"></div>
+        <h2>Help us make an impact.</h2>
+        <br>
+        <h5 style="font-weight: normal">LoopBack has an active and growing open-source community.</h5>
+        <div class="header-space"></div>
+    </div>
+    <hr class="hairline">
+
+    <!-- Contribute to LoopBack section -->
+    <section class="odd-section">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 text-center">
+                    <h3 class="section-heading" style="padding-top: 48px;">Contribute to LoopBack</h3>
+                    <p>How do you wish to contribute?</p>
+                    <p style="height:24px"></p>
+                </div>
+            </div>
+    
+            <div class="row text-center" style="justify-content: center">
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2" href="https://github.com/strongloop/loopback-next/issues">
```

Comment:
I think it should point to https://loopback.io/doc/en/contrib/Reporting-issues.html instead.